QuoteEngine/README.md

```diff
@@ -1,30 +1,177 @@
+      <br>
+      For phpmyadmin you need the following packages: apache2, php <- install those first. :)
+3. Install eggdrop. Check out here: https://www.eggheads.org/downloads
 
-Note: Give the SQL user minimum permissions, restrict it to just the host the bot runs on, etc.
+In case you have a running bot, you can continue.
 
-TCL setup
----------
 
-1. Install mysqltcl package
-2. Put the QuoteEngine.tcl file in your eggdrop's scripts/ directory
-3. Edit the QuoteEngine-settings.sample.tcl file and save it as
-   QuoteEngine-settings.tcl in your scripts/ directory
-4. Put `source scripts/QuoteEngine.tcl` in your bot's config file
+Database setup, first steps
+---------------------------------------------
+
+1. Create admin user for mysql if you don't have it yet.
+   
+   // In case you have an admin user for your mysql server other than the "root" user, you can skip this. //
+
+   With root user enter the following command in your shell:
+   <br>
+   ```# mysql -u root```
+      <br>
+      This will let you to login to mysql, so you can make changes on/in it.
+ 
+   Create a new admin user (so you can login to phpmyadmin later on):
+   <br>
+   ```mysql> CREATE USER '<user>'@'%' IDENTIFIED BY '<pwhere>';```
+      <br>
+      This will create the <user> with the given password.
+      
+   Grant all right to the user:
+   <br>
+   ```mysql> GRANT ALL PRIVILEGES ON *.* TO '<user>'@'%' WITH GRANT OPTION;```
+      <br>
+      This command allows you to login from any host/ip and gives you unlimited control over all databases.
+   
+   Check if you did everything well, so the user exists:
+   <br>
+   ```mysql> SHOW GRANTS FOR '<user>'@'%';```
+   <br>
+   ```mysql> exit```
+   <br>
+      This will exit you out from the mysql console.
+   
+2. Create a user and database in your mysql server for the bot to use.
+   
+   You have two options now:
+   <br>
+   <br>
+   a, Use phpmyadmin, where you can log in with the user which you created in point one and there
+      create a new database and a new user, then grant that user rights over the database ..
+   
+   OR
+   
+   b, Use the following commands:
+   
+   With your shell user, login to mysql with the previously created user:
+   <br>
+   ```$ mysql -u <user> -p```
+   <br>
+      Here you will need to enter the password which you have given previous (`<pwhere>`).
+   <br>
+   
+   ```mysql> CREATE DATABASE quotesdb;```
+   <br>
+      This will create a database, named "quotesdb".
+   
+   Now we need to create an other user for the bot.
+   <br>
+   ```mysql> CREATE USER '<botnick>'@'localhost' IDENTIFIED BY '<botpwhere>';```
+   <br>
+      This will create a user named "botnick". We will use this user,
+      <br>
+      to connect to mysql and make changes in database named "quotesdb".
+      <br>
+      For that, we need to set rights.
+      
+   ```mysql> GRANT ALL PRIVILEGES ON quotesdb.* TO '<botnick>'@'localhost';```
```

Comment:
👍 for localhost here

I think the privs can be something like `GRANT SELECT, INSERT, DELETE, UPDATE ON ...`

Reply:
okay. updated. although it's the bot's database.. so not a big deal i think. :)